app/models/miq_region.rb

```diff
@@ -243,6 +243,7 @@ def remote_ws_hostname
   end
 
   def remote_ws_url
+    return ::Settings.webservices.url if Rails.env.development?
```

Comment:
I think this could also be useful for central admin if we change it to be more general.

I think it would have to be something like this (keep me honest @Fryguy):

```ruby
remote_url_override = remote_ws_miq_server.settings_for_resource.webservices.url
return remote_url_override if remote_url_override
```

With this we can tell users to set the url they want to use for a particular server. I'm sure there's some networking scenario where the hostname/ip a server has external to the region is different than the one it can detect locally.